lib/seattleflu/db/cli/command/clinical.py

```diff
@@ -257,30 +252,63 @@ def sch_clinical(sch_filename):
         "sex": "AssignedSex",
         "census_tract": "census_tract",
     }
-    df = df.rename(columns=column_map)
+    clinical_records = clinical_records.rename(columns=column_map)
 
     # Drop unnecessary columns
-    df = df[column_map.values()]
-    df["encountered"] = pd.to_datetime(df["encountered"])
+    clinical_records = clinical_records[column_map.values()]
 
     # Insert static value columns
-    df["site"] = "SCH"
+    clinical_records["site"] = "SCH"
 
     #Create encounter identifier(individual+encountered)
-    df["identifier"] = (df["individual"] + \
-                        df["encountered"].astype(str)).str.lower()
+    clinical_records["identifier"] = (clinical_records["individual"] + \
+                        clinical_records["encountered"].astype(str)).str.lower()
 
     #Hash individual and encounter identifiers
-    df["individual"] = df["individual"].apply(generate_hash)
-    df["identifier"] = df["identifier"].apply(generate_hash)
+    clinical_records["individual"] = clinical_records["individual"].apply(generate_hash)
+    clinical_records["identifier"] = clinical_records["identifier"].apply(generate_hash)
 
 
     # Placeholder columns for future data
-    df["FluShot"] = None
-    df["Race"] = None
-    df["HispanicLatino"] = None
-    df["MedicalInsurace"] = None
-
-    session = DatabaseSession()
-    with session, session.cursor() as cursor:
-        df.apply(lambda x: insert_clinical(x, cursor), axis=1)
+    clinical_records["FluShot"] = None
+    clinical_records["Race"] = None
+    clinical_records["HispanicLatino"] = None
+    clinical_records["MedicalInsurace"] = None
+
+    print(clinical_records.to_json(orient='records', lines=True))
```

Comment:
Oh! Does this print out ndjson automatically? Maybe we can refactor the manifest process to do the same?

Reply:
Yep, `lines=True` prints a new JSON record per line. 
Oh yes, that is a possibility. For some reason it didn't occur to me that `manifest` also uses pandas.

Reply:
The manifest stuff doesn't use `pandas.DataFrame.to_json`, though, because I wanted streaming output (instead of being all buffered at once) and more explicit control (and less magic) over the JSON produced. For example, the reason the date-as-bigint thing was an issue is because the default is to represent dates as epoch values (seconds since 1970-01-01 00:00:00) instead of ISO 8601 strings.

So the manifest code instead defines a `dump_ndjson(iterable)` function which calls `as_json(dict)` from seattleflu.db.datatypes. This module is shared by some other code as well.

This `dump_ndjson()` could be refactored and shared by the clinical code here too.

Reply:
Would you like me to update the `parse` process here to use `dump_ndjson()`? Or should we wait to do so in the refactoring-of-shared-code PR?

Reply:
Whatever is less work for you!

Reply:
I'd rather kick that down the road to #27